lib/controllers/episodes.js

```diff
@@ -1,3 +1,23 @@
 const { Router } = require('express');
+const { Episode } = require('../models/Episode');
 
-module.exports = Router();
+module.exports = Router()
+  .get('/', async (req, res) => {
+    const eps = await Episode.getAll();
+    const filtered = eps.map(({ 
+      id,
+      title,
+      season,
+      number,  
+      quotes
+    }) =>
+      ({
+        id,
+        title,
+        season,
+        number,
+        quotes
+      }));
```

Comment:
Same issue here. You're mapping through an array and then just returning every item in the array unchanged?

Reply:
Never mind it occurred to me the object you're destructing may have more properties than you're using. It works, but it is hard to read in my opinion and I'd probably opt for something like:

```js 
const filtered = eps.map((item) =>
      ({
        id: item.id,
        title: item.title,
        season: item.season,
        number: item.number,
        quotes: item.quotes
      }));
```